src/Config/Config.ts

```diff
@@ -315,22 +316,25 @@ export class Config {
   /**
    * It provides config object from the path specified. This will throw if
    * mosaic config path or facilitator config path doesn't exists.
+   * @param facilitatorConfigPath Path to facilitator config file path.
    * @param mosaicConfigPath Path to mosaic config file path.
-   * @param facilitatorConfigPath Path to facilitator config file path/
+   * @param gatewayConfigPath Path to gateway config file path.
    * @returns Config object consisting of gateway addresses and facilitator configurations.
    */
   public static fromFile(
-    mosaicConfigPath: string,
     facilitatorConfigPath: string,
+    mosaicConfigPath?: string,
+    gatewayConfigPath?: string,
   ): Config {
-    const mosaic: MosaicConfig = MosaicConfig.fromFile(mosaicConfigPath);
     const facilitator: FacilitatorConfig = FacilitatorConfig.fromFile(facilitatorConfigPath);
+    const gatewayAddresses = mosaicConfigPath
```

Comment:
Both params mosaicConfig and gatewayConfig are optional. It should check one of both exists. 

Compiler is also complaining on same point on statement `gatewayConfigPath!` because of `!`